DEV_DOC.md

```diff
@@ -0,0 +1,88 @@
+# To add a new algorithm in PgRouting
+
+## Installation
+
+Requirements for `cmake` :
+```shell
+sudo apt update
+sudo apt install build-essential
+sudo apt install libboost-graph-dev
+sudo apt install postgresql-server-dev-14
+sudo apt install sphinx
+```
+
+Install `cmake` and `g++` :
+```shell
+sudo apt install cmake
+sudo apt install g++
+```
+
+Compile :
+```shell
+mkdir build
+cd build
+cmake -DWITH_DOC=ON ..
+make
+sudo make install
+```
+
+Pour installer l'extension : 
+```sql
+dropdb --if-exists -h localhost -U postgres -p 5432 tests_pgrouting
+createdb -h localhost -U postgres -p 5432 tests_pgrouting
+psql -h localhost -U postgres -p 5432 -d tests_pgrouting 
+> CREATE EXTENSION pgrouting CASCADE;
+> SELECT pgr_version();
+```
+
+## How modules are structured?
```

Comment:
I understand your approach, however I'm quite balanced about the example, which may be too specific in order to accurately illustrate the contribution procedure... Maybe insist on the fact it is only an example...?